changelog.md

```diff
@@ -12,6 +17,11 @@
     + inflection@1.6.0
     + lodash@3.5.0
     + validator@3.34
+    + generic-pool@2.2.0
+- [INTERNALS] Updated devDependencies.
```

Comment:
Ah alright :)
devDependencies don't matter much for a changelog, but `generic-pool` is a good add. We'll leave it in.